Find includes and libs in the  tree, install there.
Use cp instead of 'install -t', which does not work on Mac OS X.
Disable plrs build.
Use LDFLAGS


--- patches/makefile	2015-01-21 07:36:46.000000000 +0100
+++ patches/makefile	2015-12-29 19:09:22.428735582 +0100
@@ -25,8 +25,8 @@
 #Select a path below to give location of gmp library
 
 #cygwin
-INCLUDEDIR = /usr/include
-LIBDIR     = /usr/lib
+INCLUDEDIR = ${SAGE_LOCAL}/include
+LIBDIR     = ${SAGE_LOCAL}/lib
 
 #linux at mcgill with gmp version 3
 #INCLUDEDIR = /usr/local/include
@@ -45,7 +45,7 @@
 LIBRARIES=$(LIB) $(SHLIB) $(SHLINK)
 
 # where to install binaries, libraries, include files
-prefix := /usr/local
+prefix := ${SAGE_LOCAL}
 
 # Shared library
 SONAME ?=liblrsgmp.so.0
@@ -57,11 +57,11 @@
 
 # rule to build gmp arithmetic using object files.
 %-GMP.o: %.c lrsgmp.h
-	$(CC) -c $(CFLAGS) $(CPPFLAGS) -DGMP -o $@ $<
+	$(CC) -c $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) -DGMP -o $@ $<
 
 # rule to build gmp arithmetic and relocatable object files.
 %-GMP-SHR.o: %.c lrsgmp.h
-	$(CC) -c -fPIC $(CFLAGS) $(CPPFLAGS) -DGMP -o $@ $<
+	$(CC) -c -fPIC $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) -DGMP -o $@ $<
 
 # Cancel built in rules
 %: %.c
@@ -69,7 +69,7 @@
 
 # How to build a gmp arithmetic using tool
 %: %-GMP.o $(LRSGMPLIB)
-	$(CC) $< -L. -llrsgmp -L${LIBDIR}  -lgmp -o $@
+	$(CC) $< -L. -llrsgmp -L${LIBDIR} -lgmp $(LDFLAGS) -o $@
 
 all: $(BINARIES)
 
@@ -86,62 +86,73 @@
 	$(RANLIB) $@
 
 $(SHLIB): lrslib-GMP-SHR.o lrsgmp-GMP-SHR.o
-	$(CC) -shared -Wl,-soname=$(SONAME) $(SHLIBFLAGS) -o $@ $^ -lgmp
+	$(CC) -shared -Wl,-soname=$(SONAME) $(SHLIBFLAGS) $(LDFLAGS) -o $@ $^ -lgmp
 
 $(SHLINK): $(SHLIB)
 	ln -sf $< $@
 
 lrs1:	lrs.c lrslib.c lrslib.c lrslong.c
-	$(CC) $(CFLAGS) $(CPPFLAGS) -DLONG -o lrs1 lrs.c lrslib.c lrslong.c
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) -DLONG -o lrs1 lrs.c lrslib.c lrslong.c
 
 redund1: redund.c lrslib.c lrslib.c lrslong.c
-	$(CC) $(CFLAGS) $(CPPFLAGS) -DLONG -o redund1 redund.c lrslib.c lrslong.c
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) -DLONG -o redund1 redund.c lrslib.c lrslong.c
 
 setnash: setupnash.c lrslib.c lrsmp.c
-	$(CC) $(CFLAGS) $(CPPFLAGS) -o setnash setupnash.c lrslib.c lrsmp.c
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) -o setnash setupnash.c lrslib.c lrsmp.c
 
 setnash2: setupnash2.c lrslib.c lrsmp.c
-	$(CC) $(CFLAGS) $(CPPFLAGS) -o setnash2 setupnash2.c lrslib.c lrsmp.c
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) -o setnash2 setupnash2.c lrslib.c lrsmp.c
 
 install: install-static
 
 install-shared: all-shared install-common
 	 mkdir -p $(DESTDIR)${prefix}/lib
-	 install -t $(DESTDIR)${prefix}/lib $(SHLIB)
+	 cp $(SHLIB)  $(DESTDIR)${prefix}/lib 
 	 cd $(DESTDIR)${prefix}/lib && ln -sf $(SHLIB) $(SHLINK)
 	 cd $(DESTDIR)${prefix}/lib && ln -sf $(SHLIB) $(SONAME)
 
 install-static: all-static install-common
 	 mkdir -p $(DESTDIR)${prefix}/lib
-	 install -t $(DESTDIR)${prefix}/lib $(LIB)
+	 cp $(LIB) $(DESTDIR)${prefix}/lib 
 
 install-common:
 	 mkdir -p $(DESTDIR)${prefix}/bin
-	 install -t $(DESTDIR)${prefix}/bin $(BINARIES)
+	 cp $(BINARIES) $(DESTDIR)${prefix}/bin 
 	 mkdir -p $(DESTDIR)${prefix}/include
-	 install -t $(DESTDIR)${prefix}/include lrslib.h lrsgmp.h
+	 cp lrslib.h lrsgmp.h $(DESTDIR)${prefix}/include 
 
 fourier:	fourier.c lrslib.h lrslib.c lrsgmp.h lrsgmp.c
-	gcc -O3 -DTIMES -DSIGNALS  -DGMP -I${INCLUDEDIR} fourier.c lrslib.c lrsgmp.c -L${LIBDIR}  -lgmp -o fourier
+	gcc -O3 -DTIMES -DSIGNALS  -DGMP -I${INCLUDEDIR} fourier.c lrslib.c lrsgmp.c -L${LIBDIR} $(LDFLAGS) -lgmp -o fourier
 
 
 plrs:	  	plrs.cpp plrs.hpp lrslib.c lrslib.h lrsmp.c lrsmp.h
-		g++ -DGMP  -Wall -Wno-write-strings -Wno-sign-compare -I${BOOSTINC}  -Wl,-rpath=${BOOSTLIB} -O3 -DPLRS -DGMP -o plrs plrs.cpp lrslib.c lrsgmp.c -L${BOOSTLIB} -lboost_thread -lboost_system -lgmp
+		g++ -DGMP  -Wall -Wno-write-strings -Wno-sign-compare -I${BOOSTINC}  -Wl,-rpath=${BOOSTLIB} $(LDFLAGS) -O3 -DPLRS -DGMP -o plrs plrs.cpp lrslib.c lrsgmp.c -L${BOOSTLIB} -lboost_thread -lboost_system -lgmp
 
 plrsmp:	  	plrs.cpp plrs.hpp lrslib.c lrslib.h lrsmp.c lrsmp.h
-		g++ -Wall -Wno-write-strings -Wno-sign-compare -Wno-unused-variable -I${BOOSTINC}  -L${BOOSTLIB} -Wl,-rpath=${BOOSTLIB} -O3 -DPLRS -DLONG -o plrs1 plrs.cpp lrslib.c lrslong.c -lboost_thread -lboost_system
-		g++ -Wall -Wno-write-strings -Wno-sign-compare -Wno-unused-variable -I${BOOSTINC}  -L${BOOSTLIB} -Wl,-rpath=${BOOSTLIB} -O3 -DPLRS -o plrsmp plrs.cpp lrslib.c lrsmp.c  -lboost_thread -lboost_system
+		g++ -Wall -Wno-write-strings -Wno-sign-compare -Wno-unused-variable -I${BOOSTINC}  -L${BOOSTLIB} -Wl,-rpath=${BOOSTLIB} $(LDFLAGS) -O3 -DPLRS -DLONG -o plrs1 plrs.cpp lrslib.c lrslong.c -lboost_thread -lboost_system
+		g++ -Wall -Wno-write-strings -Wno-sign-compare -Wno-unused-variable -I${BOOSTINC}  -L${BOOSTLIB} -Wl,-rpath=${BOOSTLIB} $(LDFLAGS) -O3 -DPLRS -o plrsmp plrs.cpp lrslib.c lrsmp.c  -lboost_thread -lboost_system
 
 allmp:		lrs.c lrslib.c lrslib.h lrsmp.c lrsmp.h
-		gcc -Wall -O3 -DTIMES -DSIGNALS -o lrs lrs.c lrslib.c lrsmp.c
-		gcc -Wall -O3 -DTIMES -DSIGNALS -DLONG -o lrs1 lrs.c lrslib.c lrslong.c
-		gcc -O3 -DTIMES -DSIGNALS -o redund  redund.c lrslib.c lrsmp.c
-		gcc -O3 -DTIMES -DSIGNALS -DLONG -o redund1  redund.c lrslib.c lrslong.c
-		gcc -O3 -DLRS_QUIET  -DTIMES -DSIGNALS -o nash nash.c lrslib.c lrsmp.c
-		gcc -O3 -o setnash setupnash.c lrslib.c lrsmp.c
-		gcc -O3 -o setnash2 setupnash2.c lrslib.c lrsmp.c
-		gcc -O3 -o 2nash 2nash.c
-
+		gcc $(LDFLAGS) -Wall -O3 -DTIMES -DSIGNALS -o lrs lrs.c lrslib.c lrsmp.c
+		gcc $(LDFLAGS) -Wall -O3 -DTIMES -DSIGNALS -DLONG -o lrs1 lrs.c lrslib.c lrslong.c
+		gcc $(LDFLAGS) -O3 -DTIMES -DSIGNALS -o redund  redund.c lrslib.c lrsmp.c
+		gcc $(LDFLAGS) -O3 -DTIMES -DSIGNALS -DLONG -o redund1  redund.c lrslib.c lrslong.c
+		gcc $(LDFLAGS) -O3 -DLRS_QUIET  -DTIMES -DSIGNALS -o nash nash.c lrslib.c lrsmp.c
+		gcc $(LDFLAGS) -O3 -o setnash setupnash.c lrslib.c lrsmp.c
+		gcc $(LDFLAGS) -O3 -o setnash2 setupnash2.c lrslib.c lrsmp.c
+		gcc $(LDFLAGS) -O3 -o 2nash 2nash.c
+
+check:
+	lrs ext/test/cut16_11.ext > testout1
+	echo `grep -G Volume testout1` > testout1
+	echo "*Volume= 32768/14175" > testout2
+	diff testout1 testout2
+	lrs ine/test/cyclic17_8.ine > testout1
+	echo `grep -G Totals testout1` > testout1
+	echo "*Totals: vertices=935 rays=0 bases=935 integer_vertices=0" > testout2
+	diff testout1 testout2
+	rm testout1
+	rm testout2
 
 clean:		
-		rm -f $(BINARIES) $(LIBRARIES) plrs *.o *.exe
+		rm -f $(BINARIES) $(LIBRARIES) plrs *.o *.exe testout1 testout2
